### PR TITLE
newsletter: added Jan 18

### PR DESCRIPTION
**Pull request by orrabin**

### CODE REVIEW

**Review by tbrisker — COMMENTED** [review]
Perhaps we should wait for 1.17 rc1 to be out? should be announced today i think, @xprazak2 ?

**Comment by xprazak2** [comment]
It would be nice to have RC1 in newsletter, I am working on it but have no estimates on when it will be done.

File: _posts/2018-01-30-foreman-community-newsletter---january-2018.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+* [Community Demo - 25th Jan](https://www.youtube.com/watch?v=ekf8Y082HoQ)
+
+In this community demo we saw RSS notifications, fact filtering, a new plugin and tool and more.
+We also updated where to find us in upcoming conferences.
+
+* [Community Demo - 4th Jan](https://www.youtube.com/watch?v=BVTiwSEMcyw)
+
+In this community demo we saw RSS notifications, fact filtering, a new plugin and tool and more.
+We also updated where to find us in upcoming conferences.
```
Comment (xprazak2) [inline _posts/2018-01-30-foreman-community-newsletter---january-2018.md:58]: Is having the same text here and on the line 42 intentional?

Comment (orrabin) [reply _posts/2018-01-30-foreman-community-newsletter---january-2018.md:58]: nope, this was supposed to be updated :)

**Comment by orrabin** [comment]
@xprazak2 we can hold off until tomorrow morning if it will be ready today but if not I'd prefer adding it to the next newsletter.

**Comment by xprazak2** [comment]
@orrabin, it is not ready and it will take some time, I think you can go ahead and we'll add it into the next newsletter.

**Comment by GregSutcliffe** [comment]
ty @orrabin!

**Pull request merged**